orchestrator/src/indexer/rooch.ts

```diff
@@ -1,15 +1,23 @@
 import env from "@/env";
-import { instance as xTwitterInstance } from "@/integrations/xtwitter";
+import { xTwitterInstance } from "@/integrations/xtwitter";
 import { log } from "@/logger";
 import type { IEvent, IRequestAdded, JsonRpcResponse, ProcessedRequestAdded, RoochNetwork } from "@/types";
 import { decodeNotifyValueFull } from "@/util";
-import { Args, RoochClient, Secp256k1Keypair, Transaction, getRoochNodeUrl } from "@roochnetwork/rooch-sdk";
+import {
```

Comment:
Formatting here needs some help.
Please ensure Biome is installed on your IDE

Reply:
Biome is on my IDE, and we have a git action what run it the format on commit 

the import increased hence the split into multiple lines